src/_hooks/useValidateSchema.ts

```diff
@@ -17,14 +21,14 @@ export function useValidateSchema<T>(
       return;
     }
     schema
-      .validate(value, { strict: true, abortEarly, recursive }) // to avoid taking a useEffect dependency on validateOpts that is an object
+      .validate(debouncedValue, { strict: true, abortEarly, recursive }) // to avoid taking a useEffect dependency on validateOpts that is an object
       .then(() => {
         setErrors(noError);
       })
       .catch(e => {
         setErrors(e.errors || unknownError);
       });
-  }, [schema, value, abortEarly, recursive]);
+  }, [schema, debouncedValue, abortEarly, recursive]);
 
-  return errors;
+  return { errors, isStale };
```

Comment:
Should this maybe be an array, in a similar way other hooks do? `[errors, isStale]`?

Reply:
@lottamus ^

Reply:
Is there a benefit? What would that look like?

```suggestion
  return [{ errors, isStale }];
```

Reply:
It would be similar to state `[state,setState] = React.useState('')` — it's just a style thing though